lib/__tests__/dom.nested.test.js

```diff
@@ -51,7 +51,9 @@ describe('DocumentMeta - DOM nested', () => {
       <div>
         <DocumentMeta {...DOC_META} />
         <div>
-          <DocumentMeta {...DOC_META_NESTED} extend />
+          <DocumentMeta {...DOC_META_NESTED} extend>
+            <div>Hello world!</div>
```

Comment:
Sorry, I meant, nothing to the DOM that has any influence on the visual. Thus no reason the nest it.

But I'll have a look at it..